### PR TITLE
Add: Company Profile - Positiwise

### PR DESCRIPTION
**Pull request by milesbrowntech**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [x] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: company-profiles/positiwise.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Company blurb
```
Comment (KieranRobson) [inline company-profiles/positiwise.md:1]:
```suggestion
# Positiwise 

## Company blurb

```

File: company-profiles/positiwise.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+India, USA
+
+## Company technologies
+.NET, Go, React, Wordpress, WHMCS, PHP and Python
+
+## Office locations
+India, USA
+
+## How to apply
+https://positiwise.com
```
Comment (KieranRobson) [inline company-profiles/positiwise.md:20]:
```suggestion
Find job openings here: https://positiwise.com/careers
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Couple changes to be made.

File: company-profiles/positiwise.md (hunk 1 above)
Comment (milesbrowntech) [reply company-profiles/positiwise.md]: @KieranRobson this has been resolved.

File: company-profiles/positiwise.md (hunk 2 above)
Comment (milesbrowntech) [reply company-profiles/positiwise.md]: @KieranRobson this has been resolved.

File: company-profiles/positiwise.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Positiwise
+Positiwise.com now offers a wide range of services, including Custom Software Development, with expertise in both Python and .NET technologies. Our skilled team is proficient in crafting tailored solutions to meet your unique business needs. Trust us for top-notch .NET development services, backed by a proven track record of excellence in the industry.
```
Comment (KieranRobson) [inline company-profiles/positiwise.md:2]:
```suggestion
# Positiwise

## Company Blurb
Positiwise.com now offers a wide range of services, including Custom Software Development, with expertise in both Python and .NET technologies. Our skilled team is proficient in crafting tailored solutions to meet your unique business needs. Trust us for top-notch .NET development services, backed by a proven track record of excellence in the industry.

```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Company Blurb needs to be added

**Comment by milesbrowntech** [comment]
@KieranRobson Company Blurb has been added!

File: company-profiles/positiwise.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Positiwise
+
+## Company Blurb
```
Comment (KieranRobson) [inline company-profiles/positiwise.md:3]: Woops, this is meant to be a lowercase B, My mistake

```suggestion
## Company blurb
```

**Review by KieranRobson — APPROVED** [review]
Looks good to me!

**Pull request merged**